docs/config.md

```diff
@@ -1,457 +1,147 @@
->     cb_2015_24_bg_500k.prj  	cb_2015_24_bg_500k.shp.xml
->     cb_2015_24_bg_500k.shp  	cb_2an015_24_bg_500k.shx
-> maryland_blkgrps_2015.json
-> piecewise_config.json
-> README.md
-> ```
-
-#### Update the Piecewise configuration files
-
-Now we have all the information we need to configure your Piecewise server. This will involve updating the values in several files in your configuration folder.
-
-##### Edit the primary Ansible playbook
-
-The main Ansible playbook YAML file, ```piecewise/playbook.yml```, must be edited to include our new Baltimore example Ansible playbook. Lines in this file that begin with a **#** sign are code comments that highlight the modifications you need to make to the lines below the comment.
+We recommend that you open a text document to save some of this information for future reference, and to use to customize Piecewise in later steps.
 
-```
----
-# This is main Ansible playbook that configures your Piecewise server. You don't 
-# need to be an Ansible expert to use Piecewise. Follow the instructions in the 
-# code comments below to configure Piecewise for your desired location.
-#
-# For more information about Ansible and the syntax of this file, please see:
-# http://docs.ansible.com/ansible/YAMLSyntax.html
-#
-# ------------------------------------------------------------------
-# 
-# 1) Rename the folder "seattle_example" to match your folder name, and 
-#    the file "seattle_tasks.yml" to match the file name in your folder.
-#
-#    For example, change:
-#       include: seattle_example/seattle_tasks.yml
-#    to:
-#       include: baltimore_example/baltimore_tasks.yml
-
-- hosts: all
-  tasks:
-  - include: system_tasks.yml
-    sudo: True
-  - include: user_tasks.yml
-  - include: seattle_example/seattle_tasks.yml
-    sudo: True
-```
+### Select a geographic bounding box
 
-##### Edit the system tasks Ansible playbook
+We need to tell Piecewise the geographic area we're interested in. Piecewise will get M-Lab data from tests that have been run from within that area. The _bounding box_ will be in the form of four coordinates. There are many tools to help define a bounding box. We used [http://boundingbox.klokantech.com/](http://boundingbox.klokantech.com/). 
+ 
+* Enter your location in the box labelled "Find a place with Google ..." on [http://boundingbox.klokantech.com/](http://boundingbox.klokantech.com/) and press enter. Some example searches: "Baltimore, MD" or "Clearwater County, WA"
+* In the lower left menu change the settings to: **CSV**
+* Copy the coordinates at the bottom of the page into your documentation. They should like this: ```-116.4559, 46.2657, -114.5947, 46.9346```
+  * The four coordinates are the southwest and northeast corners of the area you selected. From left to right, these values are: ```<southwest longitude>,<southwest latitude,<northeast longitude>,<northeast latitude>```
+  * In your text file, put the values into the format below:
```

Comment:
Which text file? piecewise > piecewise > requirements.txt? That is the only .txt file